.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "sqltools.connections": [
+        {
+            "previewLimit": 50,
+            "server": "localhost",
+            "port": 5432,
+            "driver": "PostgreSQL",
+            "name": "photo-db",
```

Comment:
Please, don't push any credentials to the repo. If you want to share your config for extensions and etc, use env variables ([VSCODE ENV DOCS](https://code.visualstudio.com/docs/editor/variables-reference))